internal/controller/notebook_controller.go

```diff
@@ -431,14 +431,16 @@ func (r *NotebookReconciler) notebookPod(notebook *apiv1.Notebook, model *apiv1.
 		}
 	}
 
-	// Mounts specific to this Notebook.
-	if err := r.Cloud.MountBucket(&pod.ObjectMeta, &pod.Spec, notebook, cloud.MountBucketConfig{
-		Name:      "artifacts",
-		Mounts:    []cloud.BucketMount{{BucketSubdir: "artifacts", ContentSubdir: "artifacts"}},
-		Container: containerName,
-		ReadOnly:  false,
-	}); err != nil {
-		return nil, fmt.Errorf("mounting notebook: %w", err)
+	// Mounts specific to this Notebook
+	if notebook.Spec.Dataset != nil || notebook.Spec.Model != nil {
```

Comment:
Why not mount artifacts if dataset and model are not defined?

Reply:
so silly GCS fuse didn't get in the way lol. My thinking was if there is nothing to mount then we don't need to mount anything. I've reverted this back